docs/src/main/sphinx/connector/googlesheets.rst

```diff
@@ -59,15 +62,32 @@ Metadata sheet
 --------------
 
 The metadata sheet is used to map table names to sheet IDs.
-Create a new metadata sheet. The first row must be a header row
-containing the following columns in this order:
-
-* Table Name
-* Sheet ID
-* Owner
-* Notes
-
-See this `example sheet <https://docs.google.com/spreadsheets/d/1Es4HhWALUQjoa-bQh4a8B5HROz7dpGMfq_HbfoaW5LM>`_
+The first row must be a header row containing the following columns in this order (the actual header value does not matter):
+
+.. list-table:: Metadata sheet columns
+   :widths: 25 45 30
+   :header-rows: 1
+
+   * - Column
+     - Description
+     - Default
+   * - ``Table Name``
+     - Name of the table as it will show up in Trino
+     - no default - mandatory
+   * - ``Sheet ID``
+     - Sheet ID of the Google sheet. See `Sheet ID`_
+     - no default - mandatory
+   * - ``Owner``
+     - Documentation purpose only
+     - empty string
+   * - ``Notes``
+     - Documentation purpose only
+     - empty string
+   * - ``Column types``
+     - Specify types of the columns. See `Column types`_
+     - no type mapping - all columns will be ``VARCHAR``
```

Comment:
This change shouldn't belong to "Support INSERT in Google Sheets" commit. Please confirm other docs change as well.